portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,20 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Thomas O'Gara's Step Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Thomas O'Gara's Step Portfolio</h1>
+      <img src="./images/domino.jpg" alt="myself, sitting in front of a simple construction of dominos, hands held out in front of me to display my work" width="912px" height="684px">
```

Comment:
You can put these on newlines to prevent line wrapping and make it more readable:
https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping